config.yml

```diff
@@ -17,18 +17,39 @@ config:
     repository: '{{repository_user}}@{{repository_domain}}:{{namespace}}/{{project}}.git'
     repository_domain: 'github.com'
     repository_user: 'git'
+
+    dev_deploy_path: '{{deploy_root}}/{{dev_domain}}'
+    dev_domain: '{{application}}.{{dev_tld}}'
+    dev_hostname: '{{application}}-dev'
+    dev_tld: 'dev.com'
```

Comment:
Upon further research, we'll need to revise domain-related variable names & values a little. `*_tld` variables inaccurately include the Second-Level-Domain. Instead, we'll probably want to replace `*_domain` variables with `*_host` & `*_tld` variables with `*_domain` variables. We can address this issue in a separate feature branch, since these changes would also affect the Drupal recipes.
Further, we'll want to revise `*_domain` variables to use [IANA recommended](https://www.rfc-editor.org/rfc/rfc2606.html#section-2) TLD's, such as `.example` or `.invalid`.